src/Components/Patient/PatientRegister.tsx

```diff
@@ -777,6 +790,11 @@ export const PatientRegister = (props: PatientRegisterProps) => {
       name: formData.name,
       pincode: formData.pincode ? formData.pincode : undefined,
       gender: Number(formData.gender),
+      meta_info: formData.occupation
+        ? {
+            occupation: formData.occupation,
+          }
+        : null,
```

Comment:
what if there are other attributes stored in the `meta_info` object in the initial value? Those would get discarded right?

Reply:
yes @rithviknishad 

but not understanding how to handle this

Reply:
yes @rithviknishad 

but not sure how to handle this

Reply:
`{ ...initial, updated_attr: updated_val }`